<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>publicTransportStopConfig</class>
 <widget class="QWidget" name="publicTransportStopConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>138</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lblLocation">
     <property name="text">
      <string comment="@label:listbox">&amp;Country:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>location</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KComboBox" name="location">
     <property name="whatsThis">
      <string comment="@info:whatsthis">&lt;para&gt;The country the stop/station/airport is in. &lt;note&gt;Only matching service providers will be shown.&lt;/note&gt;&lt;/para&gt;
&lt;para&gt;You can also &lt;emphasis strong='1'&gt;show all available service providers&lt;/emphasis&gt; or only &lt;emphasis strong='1'&gt;international&lt;/emphasis&gt; ones.&lt;/para&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblServiceProvider">
     <property name="text">
      <string comment="@label:listbox">Service &amp;Provider:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>serviceProvider</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="KComboBox" name="serviceProvider">
       <property name="whatsThis">
        <string comment="@info:whatsthis">&lt;para&gt;Use this to &lt;emphasis strong='1'&gt;choose a service provider&lt;/emphasis&gt;, which provides data for your stop. Different service providers provide support for different cities and features.&lt;/para&gt;

&lt;para&gt;
&lt;note&gt;Some service providers that are listed in one country may also provide some data for others (limited, eg. no local public transport).&lt;/note&gt;
&lt;note&gt;To see &lt;emphasis strong='1'&gt;full information&lt;/emphasis&gt; about a service provider click on the info button on the right.&lt;/note&gt;
&lt;/para&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnServiceProviderInfo">
       <property name="toolTip">
        <string comment="@info:tooltip">Show information about the current service provider.</string>
       </property>
       <property name="text">
        <string notr="true">?</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblCity">
     <property name="text">
      <string comment="@label">&amp;City:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>city</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="KComboBox" name="city">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>location</tabstop>
  <tabstop>serviceProvider</tabstop>
  <tabstop>btnServiceProviderInfo</tabstop>
  <tabstop>city</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
